package/scripts/presto_coordinator.py

```diff
@@ -39,17 +42,22 @@ def start(self, env):
         Execute('{0} start'.format(daemon_control_script))
         if 'presto_worker_hosts' in host_info.keys():
             all_hosts = host_info['presto_worker_hosts'] + \
-                host_info['presto_coordinator_hosts']
+                        host_info['presto_coordinator_hosts']
         else:
             all_hosts = host_info['presto_coordinator_hosts']
-        smoketest_presto(
-            PrestoClient('localhost','root',
-                         config_properties['http-server.http.port']),
-            all_hosts)
+        # sleep some seconds for waiting coordinator start finished
+        time.sleep(30)
+        smoketest_presto(PrestoClient('localhost', 'root', config_properties['http-server.http.port']), all_hosts)
 
     def status(self, env):
         from params import daemon_control_script
-        Execute('{0} status'.format(daemon_control_script))
+        try:
+            Execute('{0} status'.format(daemon_control_script))
+        except ExecutionFailed as ef:
+            if ef.code == 3:
+                raise ComponentIsNotRunning("ComponentIsNotRunning")
```

Comment:
I'm a newer for submitting on github. I will take a good look at this article, and then hope to do better